charts/kubescape-cloud-operator/values.yaml

```diff
@@ -578,11 +578,11 @@ nodeAgent:
       name: node-agent
       image: 
         repository: quay.io/kubescape/sniffer
-        tag: v0.1.18
+        tag: v0.1.20
         pullPolicy: IfNotPresent
       env:  
```

Comment:
Why is all of this code in the values file and not in the DeamonSet?
Are you expecting the user to edit/change it? If not, do not write it in the values file.